test/helpers/utils.js

```diff
@@ -147,4 +147,9 @@ Utils.onchainWait = async function onchainWait (t) {
   }
 };
 
+Utils.sleep = async function sleep (seconds) {
+  seconds = seconds + 1;
+  await new Promise((resolve) => setTimeout(resolve, seconds * 1000));
```

Comment:
ah, there is a lib to move time: https://github.com/leapdao/leap-contracts/blob/master/test/helpers/time.js

Reply:
Ya 🙂 . But that doesn't work with geth 😁